community-containers/dlna/readme.md

```diff
@@ -0,0 +1,13 @@
+## DLNA server
+This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server `nextcloud-aio` in your local network.
+
+### Notes
+- This is not working on Mac OS since it requires the `host` networking mode in docker, and it doesn't really share the host's network interfaces in this system
+- If you have a firewall like ufw configured, you might need to open at least port 9999 TCP and 1900 UDP first in order to make it work.
+
```

Comment:
Should prpbably also add a note here that this is only for home networks and not for a vps for example...

Reply:
Yeah, you're right. I've added another note for that :)

Reply:
I cannot see that note yet...

Also would be good if you would keep a note like this:
`See https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers how to add it to the AIO stack`

Reply:
haha, that's because I forgot to push it :) there it goes. OK, I'll add a note you provided.

Reply:
Nice!